config.json

```diff
@@ -252,7 +269,10 @@
         "slug": "robot-name",
         "name": "Robot Name",
         "uuid": "7cb2d96a-4707-4cec-9ead-313fcb5fbb94",
-        "prerequisites": [],
+        "prerequisites": [
+          "arithmetic-operators", "structs", "methods", "randomness",
```

Comment:
What concept does `methods` refer to? The same as `multiple-dispatch` or do you have something else in mind for an exercise explicitely about methods?

Reply:
I was just picking from the list in the concepts directory. multiple-dispatch is fine, I guess I just picked it because the functions here will operate on the student-defined struct, but will probably only have the one method.

The concept matches the OOP understanding of what a method is: you don't need to know anything about multiple-dispatch or open classes to do this exercise.

But multiple-dispatch is better, wouldn't want students to think that OOP-like methods are special.

Reply:
```suggestion
          "arithmetic-operators", "structs", "multiple-dispatch", "randomness",
```